addons/l10n_sa_edi/models/account_journal.py

```diff
@@ -140,8 +147,8 @@ def _l10n_sa_get_csr_str(self):
             (NameOID.ORGANIZATIONAL_UNIT_NAME, (company_id.vat or '')[:10]),
             # Organization Name
             (NameOID.ORGANIZATION_NAME, company_id.name),
-            # Subject Common Name
-            (NameOID.COMMON_NAME, company_id.name),
```

Comment:
hmm.  No way to make this dynamic?  That we only add other things when we see there is already another one?  It is quite tricky maybe in stable.  (although, does it matter much if this is suddenly changed anyways?)